src/uproot/interpretation/objects.py

```diff
@@ -510,13 +510,25 @@ def _unravel_members(members):
 def _strided_awkward_form(awkward, classname, members, file, context):
     contents = {}
     for name, member in members:
-        if isinstance(member, AsStridedObjects):
-            cname = uproot.model.classname_decode(member._model.__name__)[0]
-            contents[name] = _strided_awkward_form(
-                awkward, cname, member._members, file, context
-            )
+        if not context["header"] and name in ("@num_bytes", "@instance_version"):
+            pass
+        elif not context["tobject_header"] and name in (
+            "@num_bytes",
+            "@instance_version",
+            "@fUniqueID",
+            "@fBits",
+            "@pidf",
```

Comment:
Yes, there's a limited set of these: "headers" are 4-byte or 6-byte things that either just have the expected number of bytes or that plus a 2-byte instance version, and "TObject headers" vary, but they usually have `fUniqueID` and `fBits`, sometimes `pidf`, and sometimes are ingested in such a way that they also include the "headers".

Even the ability to turn on `headers=True` and `tobject_headers=True` is probably on its way out, since the AwkwardForth as-written doesn't collect them and I don't think we'll ever add that ability. Deserialization code (like AwkwardForth) needs to know about them to skip over them, but if we're not including them in the arrays anymore, we might as well also remove the ability to describe them in a Form.

That would require a deprecation cycle (setting `headers=True` or `tobject_headers=True` would raise a warning), but afterward, any member starting with an "`@`" can be filtered out here, without worrying about whether it's coming from the "headers" or the "TObject headers."